src/chains/definitions/flowTestnet.ts

```diff
@@ -19,4 +19,10 @@ export const flowTestnet = /*#__PURE__*/ defineChain({
       url: 'https://testnet.flowdiver.io',
     },
   },
+  contracts: {
+    multicall3: {
+      address: '0xca11bde05977b3631167028862be2a173976ca11',
+      blockCreated: 8506,
```

Comment:
updated block height according to 
https://evm-testnet.flowscan.io/block/0xb6e4e63a28678dab9f390be4530fe4fc32a416c4df2441e507bf8d3fdbe0d581?tab=index
```suggestion
      blockCreated: 137518,
```